www/javascript/site-content-slider.js

```diff
@@ -163,6 +183,7 @@ YAHOO.util.Event.onDOMReady(function ()
 		var pos = 0;
 
 		for (var i = 0; i < this.pages.length; i++) {
+			Dom.setStyle(this.pages[i].element, 'height', 'auto');
```

Comment:
this method is getting called twice. Once `onDOMReady` when the height of the page is set (incorrectly if the dynamic image didn't load yet), and then again once the images have loaded. In this same method, on line 207, we're doing `Dom.setStyle(this.pages[i].element, 'height', max_height + 'px');` so, if we want to capture the correct height the second time, we need to reset the height to `auto`.